include/cgimap/infix_ostream_iterator.hpp

```diff
@@ -26,13 +26,13 @@ class infix_ostream_iterator {
 private:
   ostream_type *_M_stream;
   const _CharT *_M_string;
-  bool _M_first;
+  bool _M_first{true};
 
 public:
   infix_ostream_iterator(ostream_type &s)
-      : _M_stream(&s), _M_string(0), _M_first(true) {}
+      : _M_stream(&s), _M_string(0){}
```

Comment:
Add one space before {} to be in line with line 35 below?

Reply:
fixed